src/mmw/js/src/modeling/models.js

```diff
@@ -130,12 +164,32 @@ var ScenarioModel = Backbone.Model.extend({
         name: '',
         is_current_conditions: false,
         modifications: null,      // ModificationsCollection
-        active: false
+        active: false,
+        uid: null
```

Comment:
This attribute could be confused for "unique ID" when it's really for "user id".